packages/datadog-plugin-code-origin/src/lib/tags.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { getUserLandFrames } = require('../../../dd-trace/src/plugins/util/stacktrace')
+
+const limit = Number(process.env._DD_CODE_ORIGIN_MAX_USER_FRAMES) || 8
```

Comment:
Why the underscore prefix?

Reply:
It's from the RFC. Seems odd, I agree. Apparently it's been done before, though not something we've used in the Node.js tracer. It's to signify it's not to be used by end users - as in it's private and undocumented - use at your own risk. I'm working on changing it, but for now it's like this if I'm to follow the RFC.